moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -144,8 +144,10 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     current_state_monitor_ = getSharedStateMonitor(node_, robot_model_, tf_buffer_);
 
     rclcpp::Time timeout_for_servers = pnode_->now() + wait_for_servers;
```

Comment:
Doesn't `wait_for_action_server()` expect an actual timeout duration instead of a future time stamp? I thought we could just set the default value to `rclcpp::Duration(-1)` (in MGI constructor), call `client_->wait_for_action_server(wait_for_servers.toChrono())` and everyone would be happy.

Reply:
I agree that suggestion makes more sense. There is one issue. The function [to_chrono()](https://docs.ros2.org/foxy/api/rclcpp/classrclcpp_1_1Duration.html#a671704beeb2de5c6b48b4082c38a7804) is listed here but the compiler is unable to find it. I get the error: `error: no matching function for call to ‘rclcpp::Duration::to_chrono() const`.

Reply:
By default [http://docs.ros2.org/foxy/api/rclcpp_action/classrclcpp__action_1_1ClientBase.html#a725bde769458d64cd875d21b1582f589()](http://docs.ros2.org/foxy/api/rclcpp_action/classrclcpp__action_1_1ClientBase.html#a725bde769458d64cd875d21b1582f589) the timeout is infinite. I suggest we remove the `wait_for_servers` argument from the MGI constructor.